cli/src/commands/account.rs

```diff
@@ -242,6 +254,12 @@ impl Account {
             PrivateKey::try_from(seed).map_err(|_| anyhow!("Failed to convert the seed into a valid private key"))?;
         // Construct the account.
         let account = snarkos_account::Account::<N>::try_from(private_key)?;
+        // Save to file instead of printing it back to the user
+        if let Some(path) = save_to_file {
+            let mut file = File::create(path)?;
```

Comment:
I wonder if [create_new](https://doc.rust-lang.org/std/fs/struct.File.html#method.create_new) wouldn't be a bit safer, just to avoid an unintentional overwrite

Reply:
Great catch :fishing_pole_and_fish: I have updated the code and added a unit-test for this issue. We don't want to accidentally overwrite a private-key-file. :sun_with_face: